backend/clubs/serializers.py

```diff
@@ -600,9 +600,10 @@ def get_email(self, obj):
     def get_image(self, obj):
         if not obj.public:
             return None
-        if not obj.person.profile.image:
-            return None
-        image_url = obj.person.profile.image.url
+        if not obj.image:
+            if not obj.person.profile.image:
+                return None
+        image_url = obj.image.url or obj.person.profile.image.url
```

Comment:
You might want to check if object.image exists before calling obj.image.url. Otherwise calling obj.image.url would return a value error :)

Reply:
Good catch. Done